python/ccxt/base/decimal_to_precision.py

```diff
@@ -33,18 +33,24 @@
 PAD_WITH_ZERO = 6
 
 
-def decimal_to_precision(n, rounding_mode=ROUND, precision=None, counting_mode=DECIMAL_PLACES, padding_mode=NO_PADDING):
-    assert precision is not None
+def decimal_to_precision(n, rounding_mode=ROUND, numPrecisionDigits=None, counting_mode=DECIMAL_PLACES, padding_mode=NO_PADDING):
```

Comment:
only argument name changed here from `precision` to `numPrecisionDigits`, so now across all languages the argument name is same.